lib/db/DB.ts

```diff
@@ -62,14 +62,17 @@ class DB {
         {
           nodePubKey: '02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6',
           addresses: [{ host: 'xud1.test.exchangeunion.com', port: 8885 }],
+          lastAddress: { host: 'xud1.test.exchangeunion.com', port: 8885 },
```

Comment:
I don't think we should be initializing `lastAddress` like this. To me it should really mean the last address we've successfully connected with, if we haven't connected yet it should be blank.

Reply:
So, would it be okay to put host and port empty or should I make `lastAddress` field optional (which could change more parts of the code) ?